libretroshare/src/jsonapi/jsonapi.cpp

```diff
@@ -473,33 +513,57 @@ bool JsonApiServer::revokeAuthToken(const std::string& token)
 	return false;
 }
 
+bool JsonApiServer::authorizeUser(const std::string& user,const std::string& passwd)
+{
+	if(!is_alphanumeric(user) || !is_alphanumeric(passwd))
+        return false;
+
+	RS_STACK_MUTEX(configMutex);
+
+    std::string& p(mAuthTokenStorage.mAuthorizedTokens[user]);
+
+    if(p != passwd)
+	{
+        p = passwd;
+		IndicateConfigChanged();
+	}
+	return true;
+}
+
+
 bool JsonApiServer::authorizeToken(const std::string& token)
```

Comment:
Now that we have JsonApiServer::authorizeUser, we should probably delete JsonApiServer::authorizeToken or at least mark it as RS_DEPRECATED_FOR(authorizeUser) in the header